lib/tasks/bugfix.rake

```diff
@@ -58,4 +59,20 @@ namespace :bugfix do
     Template.where(org_id: funders).update_all visibility: Template.visibilities[:publicly_visible]
     Template.default.update visibility: Template.visibilities[:publicly_visible]
   end
+  
+  desc "Set all orgs.links defaults"
+  task set_org_link_defaults: :environment do
+    Org.all.each do |org|
+      org.links = '{"org":[]}'
+      org.save!
+    end
+  end
+  
+  desc "Set all template.links defaults"
+  task set_org_link_defaults: :environment do
```

Comment:
not sure if this task is needed, when the migration that adds links field to template tables run, it should update any record to the default automatically.